README.md

```diff
@@ -1,71 +1,66 @@
 # Conch API Server
 
-Datacenter build and management service
+Conch helps you build and manage datacenters.
 
-# Setup
+Conch's goal is to provide an end-to-end solution for full datacenter resource
+lifecycle: from design to initial power-on to end-of-life for all components of
+all devices.
+
+Conch is open source, licensed under MPL2.
+
+## Caveat Emptor
+
+At the time of writing, the API is not considered to be stable. While we do our
+best to prevent breakage, the core is in considerable flux and we do not
+guarantee fit or function right now. The [conch
+shell](https://github.com/joyent/conch-shell) is our current stable interface.
```

Comment:
Should we mention the ui as well as a stable interface on top?

Reply:
paging @perigrin

Reply:
the shell is our current stable interface.